src/applications/edu-benefits/0994/manifest.json

```diff
@@ -3,5 +3,20 @@
   "entryFile": "./edu-benefits-entry.jsx",
   "entryName": "0994-edu-benefits",
   "rootUrl": "/education/about-gi-bill-benefits/how-to-use-benefits/vettec-high-tech-program/apply-for-vettec-form-22-0994",
-  "production": false
```

Comment:
This app went live awhile back, https://github.com/department-of-veterans-affairs/vagov-content/pull/282/files#diff-96c9b0fc6e38eb87deabb01c5b3b2097L6. They must have just forgot to remove this.